deploy/platform/redhat/redhat_build_rpms.py

```diff
@@ -243,8 +243,11 @@ def buildRpms():
             if child.status != 0:
                 sys.stdout.flush()
                 raise Exception("Error signing rpms. status=%d" % child.status)
-        except Exception as e:
```

Comment:
This is fixed by:

https://github.com/MDSplus/mdsplus/pull/2158

Reply:
OK I skipped the commit